### DISCUSSION TITLE
I purchased AIdente pro, why didn't I receive the license code?

### DISCUSSION THREAD

**Discussion by stat11** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#594] [Q&A]
Hello, I purchased AIdente pro, why didn't I receive the license code?

**Comment by MatthiasKerbl** [comment]
Hi @stat11,

I am sorry for the inconvenience.
There are multiple reasons why you did not get your license code after the purchase:

1. Sometimes it takes up to 30 Minutes for the purchase emails to go through.
2. The purchase emails end up in the junk email folder.
3. You had a typo in the email address during your purchase.
4. Failures of the payment networks/providers.

Please send me the email address you used to purchase AlDente Pro to news@apphousekitchen.com. Afterward, I will be able to look into what went wrong.

Cheers,
Matthias

**Reply by MatthiasKerbl** [reply]
Hi there,
​
​Good news: Our payment provider Paddle could fix the technical issues with Alipay and everything should work as expected again.
​Unfortunately, they could only manage to refund all the money. You will get your money back in the next couple of workdays.
​Therefore, if you still want AlDente Pro, you need to buy AlDente Pro again and this time you will get two purchasing emails with the license code.
​
Sorry for all the inconvenience!

Cheers,
Matthias
